spec/fixtures/invalid_header_users.csv

```diff
@@ -0,0 +1,3 @@
+username, first_name,last_name,nickname,phone,email,affiliation
```

Comment:
Let's rename this fixture to `header_spaces_user.csv` since it's not actually invalid input, we just sanitize it.